### PR TITLE
Don't try to migrate `.eslintignore` if it doesn't exist

### PR DESCRIPTION
**Pull request by askoufis**
This fixes an issue where ignore file migration (handled by an eslint dep) would throw a file not found error.

### CODE REVIEW

File: packages/sku/lib/configure.js [hunk 1]
```diff
@@ -39,39 +39,44 @@ module.exports = async () => {
...
-    console.log("'.eslintignore' file detected. Attempting migration...");
+  const { shouldMigrate, eslintIgnoreExists } =
+    await shouldMigrateOldEslintConfig();
 
-    const customIgnores = migrateEslintignore({
-      hasLanguagesConfig: Boolean(languages && languages.length > 0),
-      target: paths.relativeTarget,
-    });
+  if (shouldMigrate) {
+    console.log("'.eslintignore' file detected. Attempting migration...");
```
Comment (DanDroryAu) [inline packages/sku/lib/configure.js:46]: **suggestion:** We may want to change this message.

Or move it into the second if block. We may not have an ignorefile here.

File: packages/sku/lib/configure.js [hunk 2]
```diff
@@ -39,39 +39,44 @@ module.exports = async () => {
...
       }
-
-      console.log(
-        "Please note that this is a best-effort migration and you should manually review the 'eslintIgnore' property in your sku config.",
-      );
-    } else {
-      console.log("No custom ignores found in '.eslintignore' file");
     }
 
     console.log("Deleting '.eslintrc' and '.eslintignore' files...");
```
Comment (DanDroryAu) [inline packages/sku/lib/configure.js:82]: **suggestion:** This message can be misleading

Being more explicit in what we are actually removing can be more clear. If we say we're removing your `.eslintignore` file and you have none you may not believe the next thing sku tells you ;)

**Review by DanDroryAu — COMMENTED** [review]
Couple changes regarding the logs.

File: packages/sku/lib/configure.js (hunk 2 above)
Comment (DanDroryAu) [reply packages/sku/lib/configure.js:82]: There is one right below this one too that mentions both files.

Comment (askoufis) [reply packages/sku/lib/configure.js:82]: Maybe we can cover all cases by logging "Deleting `.eslintrc` and `.eslintignore` files if they exist"?

Comment (askoufis) [reply packages/sku/lib/configure.js:82]: We probably don't even need the success message, if it throws an error we'll likely catch it elsewhere.

**Review by DanDroryAu — APPROVED** [review]
:shipit:

**Pull request merged**